src/main/java/io/vertx/ext/mail/impl/SMTPConnection.java

```diff
@@ -57,6 +60,16 @@
     socketShutDown = false;
     this.client = client;
     this.listener = listener;
+    this.nsHandler = new MultilineParser(buffer -> {
+      if (commandReplyHandler == null) {
+        log.debug("dropping reply arriving after we stopped processing \"" + buffer.toString() + "\"");
```

Comment:
we don't want to log the buffer as it enables log injection

Reply:
Done

Reply:
There were other debugging response messages as well, and I will disable those too.